src/main/webapp/i18n/en/buildQueue.json

```diff
@@ -26,7 +26,22 @@
                 "status": "Status",
                 "buildLogs": "Build Logs"
             },
-            "cancelAll": "Cancel All"
+            "cancelAll": "Cancel All",
+            "statistics": {
+                "title": "Overview",
+                "titleTooltip": "Data is fetched for the last period defined by the time span.",
```

Comment:
I would also change this translation to "Data for the specified time span is fetched" or similar. But I'm not quite sure what the original translation should intend.